docs/en/sql-reference/statements/undrop.md

```diff
@@ -0,0 +1,97 @@
+---
+slug: /en/sql-reference/statements/undrop
+sidebar_label: UNDROP
+---
+
+# UNDROP TABLE
+
+Cancels the dropping of the table.
+
+Beginning with ClickHouse version 23.3 it is possible to UNDROP a table 
+within 8 minutes of issuing the DROP TABLE statement.  Dropped tables are listed in 
+a system table called `system.dropped_tables`.
+
```

Comment:
It worth mentioning that the query works with `Atomic` databases only

Reply:
Also I've just realized that it does not undrop the inner table of an MV (for MVs without `TO` clause), it has to be undropped manually